### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by inductor**
# Description

Added PR template

## Type of change

- [x] Document update or simple typo fix

# How Has This Been Tested?

N/A

- [x] I have checked backward/forward compatibility that may cause regarding this change.

### CODE REVIEW

**Comment by mayocream** [comment]
These `Please *` sentences can be placed into annotations?
```diff
- Please include a summary of...
+ <!-- Please include a summary of... -->
```

**Comment by inductor** [comment]
@mayocream You can make "commit suggestion" and "change request" if you submit a review comment 
image

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Description
+
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+Fixes # (issue)
```
Comment (inductor) [inline .github/PULL_REQUEST_TEMPLATE.md:5]:
```suggestion
Fixes # (issue)
Sample commit suggestion
```

Comment (inductor) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: @mayocream

**Review by oke-py — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (mayocream) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: Good!

Comment (mayocream) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: Good!

**Pull request merged**